src/libs/ReportUtils.js

```diff
@@ -451,13 +451,9 @@ function getDisplayNamesWithTooltips(participants, isMultipleParticipantReport)
  * @returns {String}
  */
 function getReportName(report, personalDetailsForParticipants = {}, policies = {}) {
-    if (lodashGet(report, 'reportNameValuePairs.type') !== 'chat') {
-        return lodashGet(report, 'reportName', '');
-    }
-
```

Comment:
This was preventing this function from ever really being used. Since the report objects created in onyx from [here](https://github.com/Expensify/App/blob/b3fa99ac9585ee699adb05739dc324f836258aaa/src/libs/actions/Report.js#L222-L248) don't actually store the reportNameValuePairs. 

Every place this function is called, it's for a chatType report so I figured it's fine to just leave this check out.

Reply:
Yeah, I discovered this same bug with @srikarparsi the other day 👍